handbook/openid-connect/openid-connect.md

```diff
@@ -338,3 +367,22 @@ To resolve this issue:
 3. Modify the number at the end to a number larger than X, where X is the required size of the cache entry specified in the error log.
 
 4. Restart the NI Web Server from the `Control` tab of the NI Web Server Configuration application.
+
+### ID Token Management Encryption
+
+If the provider is configured to use an asymmmetric ID token management encryption algorithm and private keys are missing or incorrect, or if the provider is configured to use an unsupported ID token management encryption algorithm, the user will be redirected back to the SystemLink login page after logging in to the provider.
```

Comment:
Lots of good info here but kind of hard to parse. Maybe something like:

'The following situations can lead to an error and redirect the user back to the SystemLink login page after logging in to the provider:

- The provider is configured to use an asymmetric...

- The provider is configured to use an unsupported ID...'